.travis.yml

```
@@ -2,7 +2,7 @@ language: go
 
 go:
   - 1.x
-  - 1.13.x
+  - 1.14.x
```

please go directly to 1.15
```suggestion
  - 1.15.x
```